[CIE-5504] c8yedge config docs (#4146)
SoftwareAG/c8y-docs: content/edge-kubernetes/datahub.md

### Installing and using DataHub {#installing-and-using-datahub}

> DataHub is an optional component of Edge, and can be enabled by updating the `spec.dataHub` field in the Edge custom resource (CR). For more details on the `spec.dataHub` field, refer to [Edge custom resource - DataHub](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-datahub). You can also enable DataHub using `c8yedge config --set dataHub.enabled=true` command. 

Reviewer: ```suggestion
DataHub is an optional component of Edge, and can be enabled by updating the `spec.dataHub` field in the Edge custom resource (CR). For more details on the `spec.dataHub` field, refer to [Edge custom resource - DataHub](/edge-kubernetes/edge-custom-resource-definition/#k8-edge-datahub). You can also enable DataHub using the `c8yedge config --set dataHub.enabled=true` command. 
```

> For general guidance on configuring Edge, see the [Install Edge](/edge-kubernetes/installing-edge-on-k8/) and [Modify Edge](/edge-kubernetes/manage-edge/#modify-edge) sections.

Reviewer: ```suggestion
For general guidance on configuring Edge, see [Installing Edge](/edge-kubernetes/installing-edge-on-k8/) and [Modifying Edge](/edge-kubernetes/manage-edge/#modify-edge).
```

> For general guidance on configuring Edge, see [Installing Edge](/edge-kubernetes/installing-edge-on-k8/) and [Modifying Edge](/edge-kubernetes/manage-edge/#modify-edge).

Reviewer 2: ```suggestion
For general guidance on configuring Edge, see [Modifying Edge](/edge-kubernetes/manage-edge/#modify-edge).
```
I don't think Installing Edge is really relevant here
